build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
 
 
 group = 'com.techsophy.tsf'
-version = '1.2.0'
+version = '1.2.1'
```

Comment:
no need to change the version

Reply:
Okay vaibhav, will revert it back to old version.

Reply:
@VaibhavTechsophy , as per the Description of the ticket i have removed file.